src/Core/System/Snippet/Command/InstallTranslationCommand.php

```diff
@@ -21,7 +20,7 @@
  */
 #[AsCommand(
     name: 'translation:install',
-    description: 'Downloads and installs translations from the translations GitHub repository for the specified locales or all available locales',
+    description: 'Downloads and installs translations from the translations GitHub repository for the specified locales or all available locales. Re-installing will overwrite existing translations.',
```

Comment:
```suggestion
    description: 'Downloads and installs translations from the translations GitHub repository for the specified locales or all available locales. Already installed translations will be updated.',
```

Re-install suggests that it was not installed before. Guess this makes it more clear. While "overwrite" is correct, it's a destructive term. This way it doesn't sound like it can break something.

Reply:
Re-installing means that it is installed and will be installed again :) @dgrothaus-sw and i had the conclusion that overwrite is the better wording because it is technically the right behavior. I will keep the current version of the text :)

Reply:
I know what you mean, I just made it more clear to avoid unintentional interpretations. I'm fine with overwriting, just intended to avoid hesitations.